src/gateway/kong-manager/auth/reset-password.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Reset Passwords and RBAC Tokens in Kong Manager
+badge: enterprise
+---
+
+For authentication, Kong uses two different credentials for admins:
+
+1. An admin uses a **password** to log in to Kong Manager.
+2. An admin uses an **RBAC token** to make requests to the Kong Admin API.
+
+If using [basic authentication](/gateway/{{page.kong_version}}/kong-manager/auth/basic), an admin may reset their password from within Kong Manager. Since **LDAP** and **OIDC Authentication** imply that an organization stores and manages passwords outside of Kong, password reset is not possible with either type.
+
+Each RBAC token is stored in Kong as a hash. Regardless of the authentication option selected, an admin may reset their RBAC token from within Kong Manager. Note that to support confidentiality, RBAC tokens are hashed and cannot be retrieved after they are created. If a user forgets the token, the only recourse is to reset it.
+
+## Reset a forgotten password in Kong Manager
+
+### Prerequisites
+
+* `enforce_rbac = on`
+* `admin_gui_auth = basic-auth`
+* SMTP is configured to send emails
+
+### Steps
+
+1. At the login page, click **Forgot Password** beneath the login field.
+2. Enter the email address associated with the account.
+3. Click the link from the email.
+4. Reset the password. Note that you will need to provide it again immediately after the reset is complete.
+5. Log in with the new password.
+
+## Change a password from within Kong Manager
+
+### Prerequisites
+
+* `enforce_rbac = on`
```

Comment:
I think we should have a boiler plate text that sends someone to the kong.conf config page in the case that they don't know what this means. 

I'm thinking that the person that needs to change a password isn't the person that set up Kong Gateway and knows how to configure the file.

Reply:
Ack this is a terrible prerequisite. I'll adjust this.